tools/telemetry-generator/src/handlers/testPassRate.ts

```diff
@@ -11,7 +11,6 @@
  * @param logger - An ITelemetryBufferedLogger. Call its send() method to write the output telemetry events.
  */
 module.exports = function handler(fileData, logger): void {
-	// eslint-disable-next-line @rushstack/no-new-null -- External API can return an actual null
```

Comment:
Eslint isn't complaining about this anymore 🤔

Reply:
I wonder if the rule was updated to allow comparisons to null? But I wouldn't be on that.

Reply:
> The @rushstack/no-new-null rule flags type definitions with null that can be exported or used by others. The rule ignores declarations that are local variables, private members, or types that are not exported.

https://github.com/microsoft/rushstack/tree/main/eslint/eslint-plugin#rushstackno-new-null

I think it might have been added unnecessarily from the start. Adding a type that includes `null` does trigger the rule.